### ISSUE TITLE
Null pointer exception in network_manager.dart

### ISSUE THREAD

**Issue by isoos** [xvrh/puppeteer-dart#83]
https://github.com/xvrh/puppeteer-dart/blob/master/lib/src/page/network_manager.dart#L181

if `event.networkId` is null, then the `else` branch will use `requestId` as `null`.

```
Unhandled exception:
NoSuchMethodError: The getter 'value' was called on null.
Receiver: null
Tried calling: value
#0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
#1      NetworkManager._onRequestPaused (package:puppeteer/src/page/network_manager.dart:181:44)
<asynchronous suspension>
#2      _RootZone.runUnaryGuarded (dart:async/zone.dart:1314:10)
#3      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:336:11)
#4      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:263:7)
#5      _ForwardingStreamSubscription._add (dart:async/stream_pipe.dart:132:11)
#6      _MapStream._handleData (dart:async/stream_pipe.dart:232:10)
#7      _ForwardingStreamSubscription._handleData (dart:async/stream_pipe.dart:164:13)
#8      _RootZone.runUnaryGuarded (dart:async/zone.dart:1314:10)
#9      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:336:11)
#10     _BufferingStreamSubscription._add (dart:async/stream_impl.dart:263:7)
#11     _ForwardingStreamSubscription._add (dart:async/stream_pipe.dart:132:11)
#12     _WhereStream._handleData (dart:async/stream_pipe.dart:207:12)
#13     _ForwardingStreamSubscription._handleData (dart:async/stream_pipe.dart:164:13)
#14     _RootZone.runUnaryGuarded (dart:async/zone.dart:1314:10)
#15     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:336:11)
#16     _BufferingStreamSubscription._add (dart:async/stream_impl.dart:263:7)
#17     _SyncBroadcastStreamController._sendData.<anonymous closure> (dart:async/broadcast_stream_controller.dart:383:20)
#18     _BroadcastStreamController._forEachListener (dart:async/broadcast_stream_controller.dart:324:15)
#19     _SyncBroadcastStreamController._sendData (dart:async/broadcast_stream_controller.dart:382:5)
#20     _BroadcastStreamController.add (dart:async/broadcast_stream_controller.dart:250:5)
#21     Session._onMessage (package:puppeteer/src/connection.dart:228:24)
#22     Connection._onMessage (package:puppeteer/src/connection.dart:121:17)
```

**Comment by xvrh** [comment]
Hello István,
Thanks for the report. This seems easy to fix, I will try to do it quickly.
Do you have a small reproduction so I can add a test?

**Comment by isoos** [comment]
thanks for looking into it! small sample:

````dart
    final page = await browser.newPage();
    await page.setRequestInterception(true);
    page.onRequest.listen((rq) async {
      print(rq.url);
      await rq.continueRequest();
    });
    await page.goto('https://pub.dev/', wait: Until.networkIdle);
````

**Comment by xvrh** [comment]
Weird, I can't reproduce the problem. Neither on master nor on `v1.12.0`.
What is the version you are using?

**Comment by isoos** [comment]
`1.12.0` on Mac.

**Comment by isoos** [comment]
Oh, maybe it is because I've used `headless: false,` with `puppeteer.launch`? It seems to make a difference.

**Comment by xvrh** [comment]
Ok thanks I can reproduce it in headful mode. I will have a look 👍

**Comment by xvrh** [comment]
It should now be fixed in the 1.13.0 release

**Comment by isoos** [comment]
Cool, thank you!
